package-res/resources/web/pentaho/type/Context.js

```diff
@@ -57,24 +57,39 @@ define([
    * @name pentaho.type.Context
    * @class
    * @implements pentaho.type.IContext
-   * @classDesc The `Context` class contains _Value_ instance classes
+   * @classDesc A `Context` instance holds constructors/classes of _Value_ types
    * _configured_ for a particular _context_.
    *
+   * A context has several variables that define it:
+   * [container]{@link pentaho.type.Context#container},
+   * [user]{@link pentaho.type.Context#user},
+   * [theme]{@link pentaho.type.Context#theme} and
+   * [locale]{@link pentaho.type.Context#locale}.
+   * Context variables are specified at construction time, through same named keyword arguments.
+   *
+   * These variables are used to select applicable configuration rules
+   * that are applied to the types provided by the context.
+   *
+   * Because configuration rules rely on the identification of types,
+   * only non-anonymous types -
+   * those that have an [id]{@link pentaho.type.Value.Meta#id} -
+   * can be _directly_ configured.
+   *
```

Comment:
What about adding a paragraph that explains why we need a context?

It would help visualization developers understand that they should invoke the Value classes from the context instead of accessing them directly.